### PR TITLE
Facial recognition

### PR DESCRIPTION
**Pull request by adrianchen8662**
Integrates Compreface into the pipeline for processing a received frame

### CODE REVIEW

**Comment by adrianchen8662** [comment]
This push request is for Compreface integration. A couple checks and features are missing, but the baseline features are there.

File: server/settings/comprefacesettings.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "Compreface Settings": [
+        {
+            "Domain": "http://127.0.0.1"
+        },
+        {
+            "Port": "8000"
+        },
+        {
+            "API Key": "c31ce2a9-d276-417d-b7e5-bc0f3f496783"
```
Comment (AineshSootha) [inline server/settings/comprefacesettings.json:10]: This is ok to be public?

**Review by AineshSootha — APPROVED** [review]
Nice ✅

File: server/settings/comprefacesettings.json (hunk 1 above)
Comment (adrianchen8662) [reply server/settings/comprefacesettings.json:10]: Yeah it's okay to be public. The Compreface server isn't accessible from the outside, and doesn't have to be. The IP address is 127.0.0.1, which is just localhost. I should make it clear in the docs that everything runs local for this, and to not expose this port.

**Review by JustinaChano04 — APPROVED** [review]

**Pull request merged**